app-office/worklog-assistant-bin/worklog-assistant-bin-2.1.3577.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit unpacker
+
+DESCRIPTION="Hassle-free time tracking for JIRA self-hosted and OnDemand"
```

Comment:
Please use the Standard Blessed Ebuild Variable Ordering™ to make it easier for others to read it. I.e. MY_PN first, then DESC, HOMEPAGE, SRC_URI.